internal/service/service.go

```diff
@@ -21,5 +22,11 @@ func New(id principal.Signer, egressTable egress.EgressTable) (*Service, error)
 }
 
 func (s *Service) Record(ctx context.Context, nodeDID did.DID, receipts ucan.Link, endpoint *url.URL, cause ucan.Link) error {
-	return s.egressTable.Record(ctx, nodeDID, receipts, endpoint, cause)
+	if err := s.egressTable.Record(ctx, nodeDID, receipts, endpoint, cause); err != nil {
+		return err
+	}
+
+	metrics.TrackedBatchesPerNode.WithLabelValues(nodeDID.String()).Inc()
```

Comment:
Similar to my comment elsewhere, could do this here: https://github.com/storacha/etracker/blob/main/internal/db/egress/dynamodb.go#L70